go.sum

```diff
@@ -372,8 +372,8 @@ github.com/dustin/go-humanize v1.0.0 h1:VSnTsYCnlFHaM2/igO1h6X3HA71jcobQuxemgkq4
 github.com/dustin/go-humanize v1.0.0/go.mod h1:HtrtbFcZ19U5GC7JDqmcUSB87Iq5E25KnS6fMYU6eOk=
 github.com/earthly/buildkit v0.0.1-0.20220824153500-64d715da40d6 h1:oZcbJXVZsKLv+ftz3qu6fma/vCp/Z1NZvkKmaxO3l/w=
 github.com/earthly/buildkit v0.0.1-0.20220824153500-64d715da40d6/go.mod h1:Ih6/jh6JTbktkhxRID0gj6dn2GAPI6H9U6xKVxgNqBk=
-github.com/earthly/cloud-api v1.0.1-0.20220819185746-bab168ecbbf9 h1:r24RzVlEmXaJf/tX0smUEUuGYk4bkcn8TqFI5BoqR9g=
-github.com/earthly/cloud-api v1.0.1-0.20220819185746-bab168ecbbf9/go.mod h1:JhlHsW6o8zYa+XsM0nMAevRQtES35KE5R2G8tJALsnI=
+github.com/earthly/cloud-api v1.0.1-0.20220901205940-19f57fdc4d9c h1:bz2bqba5hDZuw3bEzRxgUdLtHBqaC8mO4MTOV1H4K4E=
+github.com/earthly/cloud-api v1.0.1-0.20220901205940-19f57fdc4d9c/go.mod h1:JhlHsW6o8zYa+XsM0nMAevRQtES35KE5R2G8tJALsnI=
```

Comment:
I need to update this before merging, after the PR for satellite sleep on the backend is merged.